tidy3d/plugins/adjoint/components/simulation.py

```diff
@@ -539,6 +539,7 @@ def plot(
         monitor_alpha: Optional[float] = None,
         hlim: Optional[tuple[float, float]] = None,
         vlim: Optional[tuple[float, float]] = None,
+        transpose: bool = False,
```

Comment:
All of the Adjoint examples from tidy3d-notebooks have been tested with `transpose=True`.